deltabar/deltabar_lib/deltabar_lib.py

```diff
@@ -80,10 +64,11 @@ def should_update(self, text):
 class Delta:
   def __init__(self):
     self.data = sys.modules['deltabar'].deltabar_data
+    self.track = Track()
```

Comment:
If I launch the sim with this branch, it doesn't correctly load the saved track file and thinks there's only 1 sector, because the track name/sector count aren't set right.

The Delta class is instantiated on module load (see the last line of the file), which ends up setting self.track.name and self.track.sector_count on module load -- before acMain has been called and before the sim is fully loaded -- and sim_info doesn't seem to have valid data at that point.

(If you use the hot reloader you might not notice, since it would reimport this again and then "work" on module load)

You could move this down to reinitialize_app() where the existing "track = self.track.name" is, just do self.track = Track()?